net-libs/usrsctp/Manifest

```diff
@@ -0,0 +1 @@
+DIST usrsctp-20201103.tar.gz 770742 BLAKE2B 94c7b0417a076d627065c625726eb214ec58be90d0965748610f63c076f1c172cb114dc8162af5604751cfe5e86ebf5444f002d07e43c94e461e21f6a3a4819a SHA512 a4bb4dd0ec81b2314e8590cfe5386179697e64d82915a0ef7e8f7a0baf67f88e3da89329ea73b38e8c417668f971f6580e120ed5be8588761b4b334a65ab4120
```

Comment:
I see they've made a release, but 4 years ago. Could you name this package as `0.9.3.0_p20201102` (note the date, we use a date when commit was made, not when the ebuild was written).

Reply:
will do